homeassistant/components/elkm1/__init__.py

```diff
@@ -146,24 +170,31 @@ def _included(ranges, set_to, values):
     return True
 
 
-def _create_elk_services(hass, elk):
+def _create_elk_services(hass, elks):
     def _speak_word_service(service):
+        elk = elks.get('', None)
```

Comment:
Why would there be an empty string as dictionary key? That looks weird.

Reply:
It's a dictionary of the prefixes and for backward compatibility no prefix is allowed (making the key "").  Regardless, this is a bug in that it should've been looking up prefix.